src/pages/videos/Videos.jsx

```diff
@@ -24,7 +24,7 @@ export function Videos() {
         setLoading(true);
         const { data } = await axios.get("/api/videos");
         setVideos(data.videos);
-        SetFVid(data.videos);
+        SetCatVideos(data.videos);
         const { data: data1 } = await axios.get("/api/categories");
```

Comment:
Suggestion: you could use more generic useState name as your video library won't be limited to cats.